### ISSUE TITLE
Bug with Atrule

### ISSUE THREAD

**Issue by exdeniz** [ThisIsManta/vscode-stylus-supremacy#8]
In file with postcss-css-reset
```
@reset-global pc
```
I see this:
```
Found an input node of undefined, which had a parent node of {"type":"Atrule","type":"reset-global","segments":[{"type":"Literal","val":"pc","string":"pc","prefixed":false,"lineno":1,"column":15}],"lineno":1,"column":1}
```

**Comment by ThisIsManta** [comment]
Sorry to hear that!

I'm working on something else, but I will fix it within 24 hours. Please bear with me.

**Comment by exdeniz** [comment]
It's not at all urgent, one file... trifle

**Comment by ThisIsManta** [comment]
This has been fixed at version 0.3.2. Please update the extension.

Thanks for reporting bugs as always!
